ActionButton.js

```diff
@@ -93,6 +159,45 @@ export default class ActionButton extends Component {
     ];
   }
 
+  //////////////////////
+  // ACCESSIBILITY METHODS
+  //////////////////////
+
+  announceActionButtons() {
+    const {
+      children,
+      announceActionsLabel,
+    } = this.props;
+
+    const actionButtons = filterActionButtons(children);
+
+    const actionButtonsAccessibilityAnnouncement = actionButtons.reduce((acc, actionButton) => {
+      return `${acc} ${actionButton.props.accessibilityLabel}, `;
+    }, announceActionsLabel);
+
+    if (actionButtons.length && Platform.OS === 'ios') {
```

Comment:
**Nitpick:**
Maybe you should add an `isIOS`-method to `shared.js`, that checks the that `Platform.OS === 'ios'`, and use is here